php/libraries/LorisForm.class.inc

```diff
@@ -621,6 +621,15 @@ class LorisForm
             }
         }
 
+        // check if a 'value' attribute is set for this element
+        if (!empty($this->form[$name]['value'])) {
```

Comment:
try this
if (isset($this->form[$name]['value']) && !empty($this->form[$name]['value'])) {